docs/galaxy/source/vercel.json.md

```diff
@@ -0,0 +1,39 @@
+---
+sidebar_label: 'vercel.json'
+sidebar_position: 6
+---
+# vercel.json
+
+### Array Properties
+#### 1. rewrites
+
+```
+"rewrites:[
+    {"source": "/route", "destination": "url"}
+]"
+```
+
+A rewrite is a server side routing where the page is directed to a new route but the original URL does not change.
+
+
+#### 2. redirects
+```
+"redirects:[
+    {"source": "/route", "destination": "url", "permanent": false}
+]"
+```
+
+A redirect is the opposite of rewrite where it is client sided routing and changes the original URL to its destination.
+
+
```

Comment:
On second thought, the specific patterns could be mentioned in this file. It might be possible to have a dynamic route in the source, I'm not sure but do a quick check for it :)